client/src/components/Bookings/ManageBookings.tsx

```diff
@@ -0,0 +1,17 @@
+import Box from '@material-ui/core/Box';
+import NextBooking from './NextBooking';
+import UpcomingBookings from './UpcomingBookings';
+import PastBookings from './PastBookings';
+import useStyles from './useStyles';
+const ManageBookings = (): JSX.Element => {
+  const classes = useStyles();
```

Comment:
Optional: Why don't we filter through the all bookings here and pass them down as props based on the booking date rather than having to filter through all the bookings each single time within the child components? 😄